Vungle/VungleRouter.m

```diff
@@ -128,6 +129,7 @@ - (void)initializeSdkWithInfo:(NSDictionary *)info
         
         if (placementID.length && delegateInstance) {
             [initOptions setObject:placementID forKey:VungleSDKInitOptionKeyPriorityPlacementID];
+            self.prioritizedPlacementID = placementID;
```

Comment:
Do we need to put `self.prioritizedPlacementID = [placementID copy]`, thanks ?

Reply:
Updated.